tools/tesseratos/include/tesseratos/toolbox/plugin.hpp

```diff
@@ -0,0 +1,46 @@
+/// @dir
+/// @brief @ref toolbox-plugin plugin directory.
+
+/// @file
+/// @brief Plugin entry point.
+/// @ingroup toolbox-plugin
+
+#pragma once
+
+#include <cubos/engine/cubos.hpp>
+
+namespace tesseratos
+{
+    /// @defgroup tesseratos-toolbox-plugin Toolbox
+    /// @ingroup tesseratos
+    /// @brief Adds a resource used to keep track of whether each tool is open or not.
+    ///
+    ///
+    /// ## Resources
+    /// - @ref Toolbox
```

Comment:
```suggestion
    /// - @ref Toolbox
    ///
    /// ## Dependencies
    /// - @ref imguiPlugin
```

Reply:
You should also add `- @ref toolboxPlugin` to the dependency docs of the other plugins you changed